docs/index.asciidoc

```diff
@@ -409,6 +441,22 @@ Whether to force the lowercasing of identifier fields
 
 Hash of query parameter, for example `{ "target_id" => "321" }`
 
+[id="plugins-{type}s-{plugin}-prepared_statement_bind_values"]
+===== `prepared_statement_bind_values`
+
+  * Value type is <<array,array>>
+  * Default value is `[]`
+
+Array of bind values for the prepared statement. `:sql_last_value` is a reserved predefined string
+
+[id="plugins-{type}s-{plugin}-prepared_statement_name"]
+===== `prepared_statement_name`
+
+  * Value type is <<string,string>>
+  * Default value is `""`
```

Comment:
This is set to a random string in the code, rather than blank

Reply:
Thanks, see [this comment](https://github.com/logstash-plugins/logstash-input-jdbc/pull/349#discussion_r321638743), I'm happy to leave or remove the random string and adjust the docs here depending on Colin's reply.